docs/csharp/quick-starts/toc.md

```diff
@@ -2,9 +2,11 @@
 ## [Hello world](hello-world.yml)
 ## [Numbers in C#](numbers-in-csharp.yml)
 ## [Branches and loops](branches-and-loops.yml)
+## [Interpolated strings](interpolated-strings.yml)
```

Comment:
I like putting the new tutorial in this location.

You should update the nextTutorial and nextTutorialHRef metadate in the BranchesAndLoops tutorial, and add that metadata in the new interpolated-strings tutorial.